client/components/webpack-build-monitor/index.jsx

```diff
@@ -5,7 +5,8 @@
  * External dependencies
  */
 import React from 'react';
-import { createStore } from 'redux';
+// eslint-disable-next-line wpcalypso/import-no-redux-combine-reducers
```

Comment:
It just seemed like this is living outside of "normal" Calypso store and state that using the `state/utils` `combineReducers` wasn't right. That's all geared towards adding persistence, and I don't think we'll ever want persistence on build related state.

That said, there's little or nothing to be gained by using the Redux `combineReducers`, so I went ahead and followed the linter suggestion in 2148d17.